### ISSUE TITLE
UI issue: can't change material src property because of scroll bars

### ISSUE THREAD

**Issue by Algorush** [3DStreet/3dstreet-editor#403]
Google Chrome, Win10.
I noticed this issue in the right panel. Some interface elements located on the right edge are not visible
![image](https://github.com/3DStreet/3dstreet-editor/assets/32197767/79aa43a6-e229-4262-8359-8778a281fe36)
They are visible if I disable overflow-x: hidden in dev-tools:
![image](https://github.com/3DStreet/3dstreet-editor/assets/32197767/2f7a32bc-bcd6-45a2-9655-65647cc49e51)

**Comment by kfarr** [comment]
Yes I can reproduce too on Mac OS if you choose Settings > Appearance > Show scroll bars > Always
